action-server/actions/lib/phone_number_validation.py

```diff
@@ -25,13 +26,21 @@
 logger = logging.getLogger(__name__)
 
 
-def generate_validation_code() -> str:
+def _generate_validation_code() -> str:
     return str(randrange(10 ** VALIDATION_CODE_LENGTH)).zfill(VALIDATION_CODE_LENGTH)
 
 
 async def send_validation_code(
-    phone_number: str, language: str, first_name: str, validation_code: str
-) -> bool:
+    phone_number: str, language: str, first_name: str
+) -> Optional[str]:
+
+    print(ClientSession)
+
+    if phone_number[4:7] == "555":
```

Comment:
should we assert phone_number length before ?

Reply:
I guess it would be a good idea. No typing can enforce this so even if we ensure the length somewhere else now, someone could break it.